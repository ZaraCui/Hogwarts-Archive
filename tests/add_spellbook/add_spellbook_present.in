ADD COLLECTION tests/data/books_small.csv
ADD SPELLBOOK tests/data/books_partial.csv 2002
LIST ALL
EXIT

