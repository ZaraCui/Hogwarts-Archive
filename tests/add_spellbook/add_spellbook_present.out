user: 12 spellbooks successfully added.

user: Successfully added: Beasts & Care (Newt Scamander).

user: Levitation Charms (Helga Hufflepuff)
Dark Arts Basics (Salazar Slytherin)
Patronus Guide (Godric Gryffindor)
Transfiguration I (Minerva McGonagall)
Advanced Charms (Filius Flitwick)
Herbology Field Notes (Newt Scamander)
Polyjuice Manual (Libatius Borage)
Transfiguration i (Minerva McGonagall)
patronus guide (Godric Gryffindor)
Dark Arts Basics (Salazar Slytherin)
Levitation Charms (Helga Hufflepuff)
Advanced Charms (Filius Flitwick)
Beasts & Care (Newt Scamander)

user: Ending Archive process.
